### PR TITLE
Integrate opa-client-go, switch service to using region-aware bucket, disable excessive logging

### PR DESCRIPTION
**Pull request by todor-videv1**
# Description

## What

This PR intends to improve and polish the integration of Open Policy Agent (OPA) in content-rw-neo4j.
It introduces the following changes:

1. Add opa-client-go and use it as the mechanism for communicating between the service and its OPA sidecar.
2. Change the deployment config and make the service use the new region-aware S3 buckets.
3. Stop excessive logging by disabling decision logs and tweaking the log level of the sidecar container.

## Why

[JIRA](https://financialtimes.atlassian.net/browse/UPPSF-4866)

## Scope and particulars of this PR (Please tick all that apply)

- [x] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [ ] Feature
- [ ] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

### CODE REVIEW

File: content/content_service_test.go [hunk 1]
```diff
@@ -7,6 +7,7 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
+	"github.com/Financial-Times/opa-client-go"
```
Comment (vlasakievft) [inline content/content_service_test.go:10]:
```suggestion
	opa "github.com/Financial-Times/opa-client-go"
```

**Review by ilian2233 — DISMISSED** [review]
Very good first integration of the opa library in a service. Good job!

**Review by antoanFT — APPROVED** [review]
gg

File: content/content_service_test.go (hunk 1 above)
Comment (todor-videv1) [reply content/content_service_test.go:10]: good idea, I'll integrate it

**Review by antoanFT — APPROVED** [review]
gg

**Pull request merged**